execution.bs

```diff
@@ -5007,6 +5007,25 @@ enum class forward_progress_guarantee {
             [*Note:* The `transfer` algorithm is unique in that it ignores the execution domain of
             its predecessor, using only its destination scheduler to select a customization. *--end note*]
 
+    11. <pre highlight="c++">
+        template&lt;<i>callable</i> Fun>
+          requires is_nothrow_move_constructible_v&lt;Fun>
+        struct <i>emplace-construct</i> { // exposition only
```

Comment:
_`emplace-construct`_ doesn't indicate the invocaton of the functor. Would it be better to use _`emplace-invoke`_, _`emplace-from-invoke`_, or something like?